src/components/SlateEditor/plugins/framedContent/index.tsx

```diff
@@ -49,28 +50,27 @@ const normalizerConfig: NormalizerConfig = {
   },
 };
 
-export const bodyboxSerializer: SlateSerializer = {
+export const framedContentSerializer: SlateSerializer = {
   deserialize(el: HTMLElement, children: (Descendant | null)[]) {
     if (el.tagName.toLowerCase() !== 'div') return;
     if (el.className === 'c-bodybox') {
-      return slatejsx('element', { type: TYPE_BODYBOX }, children);
+      return slatejsx('element', { type: TYPE_FRAMED_CONTENT }, children);
     }
   },
   serialize(node: Descendant, children: JSX.Element[]) {
-    if (!Element.isElement(node) || node.type !== TYPE_BODYBOX) return;
-    return <div className="c-bodybox">{children}</div>;
+    if (!Element.isElement(node) || node.type !== TYPE_FRAMED_CONTENT) return;
+    return <FramedContent>{children}</FramedContent>;
```

Comment:
Dette vil serialisere HTML'en med et emotion-klassenavn. Det kan vi ikke gjøre. Må enten kjøre en migrering i backend, beholde `div className="c-bodybox>...` her, eller så må vi gjøre en endring i article-converter som sørger for at divs med `data-type="framed-content"` eller klasesnavn `c-bodybox` rendres som en bodybox.